Firmware/USMArduinoFirmware/USMArduinoFirmware.ino

```diff
@@ -739,48 +735,36 @@ void scanI2CBus()
   Serial.println(F("Scanning for devices on the I2C bus..."));
 
   // Scan for MCP's
-  uint8_t mcp = 0;
-  for (uint8_t addr = 0; addr < sizeof(MCP_I2C_ADDRESS); addr++)
+  for (uint8_t mcp = 0; mcp < MCP_COUNT; mcp++)
   {
     Serial.print(F(" - 0x"));
-    Serial.print(MCP_I2C_ADDRESS[addr], HEX);
+    Serial.print(MCP_I2C_ADDRESS[mcp], HEX);
     Serial.print(F("..."));
 
     // Check if there is anything responding on this address
-    Wire.beginTransmission(MCP_I2C_ADDRESS[addr]);
+    Wire.beginTransmission(MCP_I2C_ADDRESS[mcp]);
     if (Wire.endTransmission() == 0)
     {
-      if (mcp < MCP_MAX_COUNT) 
```

Comment:
No longer need this check, we scan the full list of addresses defined in `config.h` - with each position associated with 16 inputs, i.e. 1-16, 17-32, 33-48 etc. A missing MCP will mean the inputs for that address will not be used.